examples/Sqlpp11/CMakeLists.txt

```diff
@@ -1,19 +1,19 @@
-# Copyright (c) 2016-2017, Ruslan Baratov
-# All rights reserved.
-
-cmake_minimum_required(VERSION 3.0)
-
-# Emulate HunterGate:
-# * https://github.com/hunter-packages/gate
-include("../common.cmake")
-
-project(download-sqlpp11)
-
-# DOCUMENTATION_START {
-hunter_add_package(Sqlpp11)
-find_package(Sqlpp11 CONFIG REQUIRED)
-
-add_executable(example example.cpp)
-target_link_libraries(example sqlpp11)
-# DOCUMENTATION_END }
-
+# Copyright (c) 2016-2017, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.5)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-sqlpp11)
+
+# DOCUMENTATION_START {
+hunter_add_package(Sqlpp11)
```

Comment:
non-blocking: Do you know what exactly changed in that file that every line is touched by the diff? Just curious. If it is the line ending changed from `\r\n` to `\n` then this is a very welcome change

Reply:
I used "git grep ... | xargs sed -i ...." under a git console on git. Even there it changes \r\n to \n.